MineSharp.Bot/Plugins/WindowPlugin.cs

```diff
@@ -450,7 +486,10 @@ private Task HandleOpenWindow(OpenWindowPacket packet)
 
         windowInfo = windowInfo with { Title = packet.WindowTitle };
         Logger.Debug($"Received Open Window Packet id={packet.WindowId}");
-        OpenWindow(packet.WindowId, windowInfo);
+        Task.Delay(100).ContinueWith(t =>
```

Comment:
Is there a reason why we wait 100ms before opening the window?

Reply:
i think i sent a video on the discord addressing that temporal change you may want to watch it, i still didn't found the reason of why the task never executes and always times out, but adding a delay make it works.

Reply:
hm seems to be a race condition. PR #71 addresses a lot of bugs regarding race conditions, maybe it fixes it.